CHANGELOG.md

```diff
@@ -4,16 +4,12 @@
 - Changes are listed under the (pre)release in which they first appear. Subsequent releases include changes from previous releases.
 - "Breaking changes" listed under a version may require action from end users or external maintainers when upgrading to that version.
 - Do not edit this file directly. This file is auto-generated using [changie](https://github.com/miniscruff/changie). For details on how to document a change, see [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-changelog-entry)
-
```

Comment:
@emmyoop does this look right that these newlines are being removed?

Reply:
It's weird because they're in the CHANGELOGS for other versions/`.latest` branches.  The file will render the same either way so it's not significant.